content/it/api-gateway.md

```diff
@@ -0,0 +1,17 @@
+---
+title: API Gateway
+status: Completed
+category: tecnologia
+tags: ["networking", "", ""]
```

Comment:
Which one would be great here, between:
- "networking"
- the Italian term of "networking"

Preview: https://deploy-preview-1092--cncfglossary.netlify.app/it/api-gateway/
![image](https://user-images.githubusercontent.com/46767780/179693162-ac1cf806-bc7d-4de3-81ad-f8cfaba8f78a.png)

@akelity PTAL!

---

For anyone's information:
the "Tags" page: https://deploy-preview-1092--cncfglossary.netlify.app/it/tags/
![image](https://user-images.githubusercontent.com/46767780/179693611-9b1244af-a5ff-41b0-a084-50030ce018dc.png)

Reply:
Networking (in the technical field) is a term commonly used in Italy